src/components/OfferCheckoutCard/OfferCheckoutCard.tsx

```diff
@@ -19,7 +25,10 @@ import {
 } from './OfferCheckoutCardStyled';
 import Price from '../Price';
 
-const OfferCheckoutCard = ({ isDataLoaded, t }) => {
+const OfferCheckoutCard = ({
+  isDataLoaded = true,
```

Comment:
just wonder if this is still needed. We could check if offer was fetched (directly from store), and then we could remove this prop